README.md

```diff
@@ -98,12 +91,61 @@ Install the unarchived tar.gz file
 ansible-galaxy collection install cisco-aci-1.0.0.tar.gz —-force
 ```
 
-### See Also
+## Use Cases
+
+Once the collection is installed, you can use it in a playbook by specifying the full namespace path to the module, plugin and/or role.
+
+### Adding a new EPG
+
+```yml
+- hosts: aci
+  gather_facts: no
+
+  tasks:
+  - name: Add a new EPG
+    cisco.aci.aci_epg:
+      hostname: apic
+      username: admin
+      password: SomeSecretPassword
+      tenant: production
+      ap: intranet
+      epg: web_epg
+      description: Web Intranet EPG
+      bd: prod_bd
+    delegate_to: localhost
```

Comment:
Do we want to delegate to localhost in this example, don't think we require this anymore with httpapi plugin. Should we make a mention of this plugin in the readme?

Reply:
It is mentioned in Optimizing Playbooks but I can add a short summary like you mentioned in the other comment.

Reply:
I will keep the delegate option here because this is what we list in all the module examples currently.